helm-charts/example-apiserver/templates/konk-service.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: konk.infoblox.com/v1alpha1
 kind: KonkService
 metadata:
-  name: {{ include "example-apiserver.fullname" . }}-api
+  name: {{ .Release.Name }}-api
```

Comment:
The pod names were getting pretty long. kubectl pod is now named `drew-apiserver-api-konk-service-kubectl-5b6dcc84b6-2tkmt`

Reply:
-api is required b/c the release names have to be unique across all charts managed by helm-operator(s)